server/pulp/server/managers/repo/unit_association_query.py

```diff
@@ -368,10 +367,7 @@ def _associated_units_by_type_cursor(unit_type_id, criteria, associated_unit_ids
         sort = criteria.unit_sort
 
         if sort is None:
-            try:
-                unit_key = units_controller.get_unit_key_fields_for_type(unit_type_id)
-            except ValueError:
-                unit_key = None
+            sort = [('_id', SORT_ASCENDING)]
```

Comment:
This loss of functionality was discussed here: https://github.com/pulp/pulp/pull/1898

The old way was also modified, resulting in the conflict.